eng/common/pipelines/templates/steps/get-pr-owners.yml

```diff
@@ -17,19 +21,8 @@ steps:
         --kusto-url-var KUSTO_URL `
         --kusto-database-var KUSTO_DB `
         --kusto-table-var KUSTO_TABLE `
-        --identity "$(Build.QueuedBy)"
-
-      $resolvedIdentity = ""
-      try { $resolvedIdentity = $result[-1] | ConvertFrom-Json } catch {}
-
-      if($resolvedIdentity) {
-        Write-Host $resolvedIdentity
-
-        Write-Host "##vso[task.setvariable variable=${{ parameters.TargetVariable }}]$($resolvedIdentity.GithubUserName)"
-      }
-      else {
-        Write-Host "Unable to locate a github user for identity $(Build.QueuedBy)"
-      }
+        --identity "$(Build.QueuedBy)" `
```

Comment:
You could eliminate the backticks if you switched this pwsh script from `|` to `>` since it is essentially only a single line that would work.

Reply:
You're right. The backticks really assist readability for me, so I'd prefer to leave them.